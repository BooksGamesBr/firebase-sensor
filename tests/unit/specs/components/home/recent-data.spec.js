import { shallowMount } from '@vue/test-utils'
import RecentData from '@/components/home/recent-data'
import { FakeRooms, FakeCategories } from '../../../data/fake-data'

const propsData = {
  rooms: {
    room1: FakeRooms[0]
  },
  categories: FakeCategories
}

describe('RecentData.vue', () => {
  it('shows a page', () => {
    const wrapper = shallowMount(RecentData, {
      propsData
    })
    wrapper.vm.currentTimeLimit = 1514764800
    expect(wrapper.html()).toMatchSnapshot()
  })
})
