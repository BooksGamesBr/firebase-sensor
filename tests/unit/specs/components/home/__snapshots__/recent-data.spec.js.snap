// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentData.vue shows a page 1`] = `
<div class="row">
  <div class="col-xs-12 col-sm-6 col-md-4">
    <recent-data-panel-stub timelimit="1544895431" categories="[object Object],[object Object]" idroom="room1" room="[object Object]"></recent-data-panel-stub>
  </div>
</div>
`;
